src/api/TencentMeeting.ts

```diff
@@ -353,5 +353,41 @@ export async function getRecordURLs(meetingRecordId : string) {
   return res;
 }
 
+export async function countOngoingMeeting() {
```

Comment:
My point was why not just use the existing `listMeetings` function? There is no need to fine tune zRes just because we don't need some fields.